lib/media/ttml_text_parser.js

```diff
@@ -217,6 +208,25 @@ shaka.media.TtmlTextParser.getLeafNodes_ = function(element) {
 
 
 /**
+ * Insert \n where <br> tags are found
```

Comment:
Strictly speaking it's not actually replacing tags. How about "Insert '\n' characters into <br> tags"